modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.inl

```diff
@@ -57,6 +57,9 @@ MechanicalMatrixMapper<DataTypes1, DataTypes2>::MechanicalMatrixMapper()
     :
       d_forceFieldList(initData(&d_forceFieldList,"forceFieldList","List of ForceField Names to work on (by default will take all)")),
       l_nodeToParse(initLink("nodeToParse","link to the node on which the component will work, from this link the mechanicalState/mass/forceField links will be made")),
+      l_stopAtNodeToParse(initData(&l_stopAtNodeToParse,false,"stopAtNodeToParse","Boolean to choose whether forceFields in children Nodes of NodeToParse should be considered.")),
```

Comment:
I look at it and wonder what "parsing" is it talking about. Parsing means 'syntax analysis'. Shouldn't it be "process" the proper term ?

Reply:
You are right. However this Boolean refers to the link "nodeToParse", so I would rather keep the name as is for now. In a future PR, these names should be changed.